five_tips_about_testable_golang_code.md

```diff
@@ -0,0 +1,171 @@
+# How to write untestable code in golang
```

Comment:
I would really emphasize the `UN` in untestable. Go big `UNtestable` so people get it right away